ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -4,18 +4,65 @@ import {
   expect,
   Locator,
 } from '@playwright/test';
-import { clickInTheMiddleOfTheScreen, pressButton } from '@utils/clicks';
+import {
+  clickInTheMiddleOfTheScreen,
+  clickOnAtom,
+  pressButton,
+} from '@utils/clicks';
 import { ELEMENT_TITLE } from './types';
-import { TopPanelButton, waitForRender } from '..';
-import { selectTopPanelButton } from './tools';
+import {
+  AtomButton,
+  DELAY_IN_SECONDS,
+  RingButton,
+  STRUCTURE_LIBRARY_BUTTON_NAME,
+  TemplateLibrary,
+  TopPanelButton,
+  selectRing,
+  waitForRender,
+} from '..';
+import { selectAtomInToolbar, selectTopPanelButton } from './tools';
 import { getLeftTopBarSize } from './common/getLeftTopBarSize';
 import { emptyFunction } from '@utils/common/helpers';
 
 export async function drawBenzeneRing(page: Page) {
-  await page.getByRole('button', { name: 'Benzene (T)' }).click();
+  await selectRing(RingButton.Benzene, page);
+  await clickInTheMiddleOfTheScreen(page);
+}
+
+export async function drawCyclohexaneRing(page: Page) {
+  await selectRing(RingButton.Cyclohexane, page);
+  await clickInTheMiddleOfTheScreen(page);
+}
+
+export async function drawCyclopentadieneRing(page: Page) {
+  await selectRing(RingButton.Cyclopentadiene, page);
   await clickInTheMiddleOfTheScreen(page);
 }
 
+export async function selectUserTemplatesAndRename(
```

Comment:
The name of this function is misleading. It doesn't perform any renaming. And even if it does, it would be logical to have new name as parameter? 
Also, i propose to have a `page` as a first argument of any function

Reply:
Done.